src/entities/trades/0x/utils.ts

```diff
@@ -0,0 +1,28 @@
+import Decimal from 'decimal.js-light'
+
+import { Percent } from '../../fractions/percent'
+import { Platform } from '../../platforms-breakdown'
+import { CODE_TO_PLATFORM_NAME } from './constants'
+import { ApiSource } from './types'
+
+export const decodePlatformName = (apiName: string): string => CODE_TO_PLATFORM_NAME[apiName] || apiName
+
+export const decodeStringToPercent = (value: string, isStringPercent?: boolean): Percent => {
+  const proportion = new Decimal(value)
+  const denominator = new Decimal('10').pow(proportion.decimalPlaces())
+  const percent = isStringPercent ? new Decimal('10').pow(2) : 1
```

Comment:
can't we make the isStringPercent into a function? What's the pattern for it to be a percentage?
Would make the arguments cleaner.

Reply:
We receive value as the string, so  0,0072 could be a percentage or just a fraction. So if it is a percentage we need to multiply the denominator by 100, that's why we have this 13th line and we will receive 0,0072%. If the value is the fraction, we want to make a percentage from it, so the denominator is multiplied only by 1 and we receive 0,72%. We get the Percent interface at the end. Code is from our older implementation, only expand by isStringPercent and it is in a separate method. I'm not sure if we want to separate to another function, I think currently it is the cleanest

Reply:
A bit separate code, to two methods. Please, check it :)